packages/core-data/src/reducer.js

```diff
@@ -628,6 +628,12 @@ export function registeredPostMeta( state = {}, action ) {
 	return state;
 }
 
+export function templateAutoDraftId( state = {}, action ) {
```

Comment:
Why do we need this, we don't need it for any other entity?

Reply:
You mean auto-drafts? I don't think we create them client-side for any other entity, no. We do need auto-drafts here so we can show an editor when you click on the root canvas. Perhaps we can refactor this away if we change the UI a bit.

Reply:
before loading the editor in JS. We call `saveEntityRecord` with `auto-draft` or something and the returned id is loaded in the editor.